data/Makefile-prepare-data.jinja2

```diff
@@ -13,7 +13,7 @@ download: {{ src_shapefile_zips }}
 
 {% for shapefile in shapefiles %}
 {{ shapefile.src_zip }}:
-	wget '{{ shapefile.url }}' -O {{ shapefile.src_zip }}
+	wget --quiet '{{ shapefile.url }}' -O {{ shapefile.src_zip }}
```

Comment:
Add print out to say we're doing **something**, and add comment saying the `--quiet` is to be friendlier with Vagrant.